core/constant/constant.go

```diff
@@ -0,0 +1,37 @@
+package constant
+
+// configuration
```

Comment:
Actually I don't think we need a universal constant package here. It might be better to put these constants to their respective packages (to make it clear)?

Reply:
> Actually I don't think we need a universal constant package here. It might be better to put these constants to their respective packages (to make it clear)?

I agree,  putting all the constants in one package is a bit radical. I will rearrange related code.